ticdc/manage-ticdc.md

```diff
@@ -594,3 +693,21 @@ To create a cyclic replication task, take the following steps:
 + To perform online DDL operations, ensure the following requirements are met:
     - The TiCDC components of multiple clusters form a one-way DDL replication chain, which is not cyclic. For example, in the example above, only the TiCDC component of cluster C disables `sync-ddl`.
     - DDL operations must be performed on the cluster that is the starting point of the one-way DDL replication chain, such as cluster A in the example above.
+
+## Output the historical value of a Row Changed Event <span class="version-mark">New in v4.0.5</span>
+
+> **Warning:**
+>
+> Currently, outputting the historical value of a Row Changed Event is still an experimental feature. It is **NOT** recommended to use it in the production environment.
+
+Before v4.0.5, in the default configuration, the Row Changed Event of TiCDC Open Protocol output in a replication task only contains the changed value, not the value before the change. Such a behavior neither supports the [new collation framework](/character-set-and-collation.md#new-framework-for-collations) introduced in TiDB v4.0 nor meets the need that the consumer ends of TiCDC Open Protocol use the historical value of a Row Changed Event.
```

Comment:
seems don't need to add `Before v4.0.5` here, both in version less than 4.0.5 and 4.0.5 have this default behavior

Reply:
removed in https://github.com/pingcap/docs/pull/3752/commits/510a8442b9d1e91b68e2ef13f46911a9988afb8c